README.rst

```diff
@@ -90,6 +90,8 @@ before running the server.
 
 - To build all styles and js for production, including semantic UI::
 
+    npm install
+    npm install -D
```

Comment:
Why is the `install -D` needed? 

We might want to adjust the language a little here. Funny we missed including the installation step!

Reply:
it's not needed; this is my miscommunication. npm installs `devDependencies` by default, so `npm install` is sufficient to install everything in `package.json`.